test/performance/range/views/view_translate_2D_1D_benchmark.cpp

```diff
@@ -41,34 +40,34 @@ void sequential_read_impl(benchmark::State & state, rng_t && rng)
 {
     for (auto _ : state)
         for (auto && outer : rng)
-            for (aa27 inner : outer)
+            for (seqan3::aa27 inner : outer)
                 benchmark::DoNotOptimize(inner.to_rank());
 }
 
 template <typename tag_t>
 void sequential_read(benchmark::State & state)
 {
-    std::vector<std::vector<dna4>> dna_sequence_collection;
+    std::vector<std::vector<seqan3::dna4>> dna_sequence_collection;
     dna_sequence_collection.resize(1000);
 
     for (size_t i = 0; i < dna_sequence_collection.size(); ++i)
-        dna_sequence_collection[i] = test::generate_sequence<seqan3::dna4>(100, 0, 0);
+        dna_sequence_collection[i] = seqan3::test::generate_sequence<seqan3::dna4>(100, 0, 0);
 
     if constexpr (std::is_same_v<tag_t, baseline_tag>)
     {
-        std::vector<std::vector<aa27>> translated_aa_sequences = dna_sequence_collection
-                                                               | views::translate_join
-                                                               | views::to<std::vector<std::vector<aa27>>>;
+        std::vector<std::vector<seqan3::aa27>> translated_aa_sequences = dna_sequence_collection
+                                                            | seqan3::views::translate_join
+                                                            | seqan3::views::to<std::vector<std::vector<seqan3::aa27>>>;
```

Comment:
misaligned :)

Reply:
The lines would get to wide, if i align it futher to the right. Do you have a better idea to split the line?

Reply:
```
        std::vector<aa27_vector> translated_aa_sequences = dna_sequence_collection
                                                         | seqan3::views::translate_join
                                                         | seqan3::views::to<std::vector<aa27_vector>>;
```

In this case we can use `aa27_vector` instead of `std::vector<aa27>`